index.bs

```diff
@@ -128,7 +140,25 @@ This callback will be invoked when there are changes to <i>target</i>'s intersec
     </pre>
 
     <div dfn-type="method" dfn-for="IntersectionObserver">
-        : <dfn>observe()</dfn>
+        : <dfn>Constructor(callback, options)</dfn>
+        ::
+            <ol>
+                <li>Let <i>this</i> be a new {{IntersectionObserver}}
+                    object</li>
+                <li> Set <i>this</i>'s internal {{[[callback]]}} slot to
+                    <i>callback</i>.</li>
+                <li> Set <i>this</i>'s internal {{[[root]]}} slot to
+                    <i>options</i>.{{root}}</li>
+                <li> Set <i>this</i>'s internal {{[[rootMargin]]}} slot to
+                    <i>options</i>.{{rootMargin}}</li>
+                <li> Set <i>this</i>'s internal {{[[threshold]]}} slot to
+                    <i>options</i>.{{threshold}}</li>
+                <li>Append <i>this</i> to the
+                    <a>unit of related similar-origin browsing contexts</a>'s
```

Comment:
FYI filed https://github.com/whatwg/dom/issues/88 to figure out if this should be more precise. No need to block this PR on such deliberations though.